lxml-stubs/etree.pyi

```diff
@@ -493,12 +493,25 @@ def cleanup_namespaces(
     keep_ns_prefixes: Optional[Iterable[_AnyStr]] = ...,
 ) -> None: ...
 def parse(
-    source: _FileSource, parser: XMLParser = ..., base_url: _AnyStr = ...
+    source: _FileSource,
+    parser: Union[XMLParser, HTMLParser] = ...,
+    base_url: _AnyStr = ...,
 ) -> _ElementTree: ...
```

Comment:
`parse()` has the same behaviour as `fromstring()`, the only difference is the type of input. It can also return "anything".
(BTW, "Any" is really correct here. A parser `target` can build and return whatever it wants – but that requires an appropriately configured parser argument.)

```suggestion
) -> Union[_ElementTree, Any]: ...
```